.ci/proj_jjb.yaml

```diff
@@ -0,0 +1,118 @@
+- job-template:
+    name: "{jjb_proj}"
+    project-type: pipeline
+    properties:
+        - github:
+             url: "{jjb_git}"
+        - build-discarder:
+            days-to-keep: 50
+            num-to-keep: 20
+        - inject:
+            keep-system-variables: true
+            properties-content: |
+              jjb_proj={jjb_proj}
+    description: Do NOT edit this job through the Web GUI !
+    concurrent: false
+    parameters:
+        - string:
+            name: "sha1"
+            default: master
+            description: "Commit to be checked, set by PR"
+        - bool:
+            name: "build_dockers"
+            default: false
+            description: "Rebuild docker containers. Check this box if .ci/DockerFile* was changed"
+        - string:
+            name: "conf_file"
+            default: ".ci/matrix_job.yaml"
+            description: "Regex to select job config file. Do not change it"
+        - string:
+            name: "DEBUG"
```

Comment:
It's better to move service params to the bottom. I mean "build_dockers", "conf_file" and "DEBUG" which are all for DevOps only.

Reply:
i see differently